README.md

```diff
@@ -22,4 +22,4 @@ In this task, we use the [Bulma CSS framework](https://bulma.io/). It is already
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_goods-selector-js/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://dmytro-voronin.github.io/react_goods-selector-js/) and add it to the PR description.
```

Comment:
The demo link doesn't work. Fix it and add to the PR description
<img width="745" alt="image" src="https://github.com/mate-academy/react_goods-selector-js/assets/98764641/2f2d5bae-f9d7-4764-a0ef-0b3f3e4a8d29">